src/mapgen.cpp

```diff
@@ -589,13 +589,13 @@ class jmapgen_alternativly : public jmapgen_piece {
     // PieceType, they *can not* be of any other type.
     std::vector<PieceType> alternatives;
     jmapgen_alternativly() = default;
-    void apply( map &m, const size_t x, const size_t y, const float mon_density ) const override
+    void apply( map &m, const size_t x, const size_t y, const size_t xmin, const size_t ymin, const size_t xmax, const size_t ymax, const float mon_density ) const override
```

Comment:
Perhaps it would be better to forward the whole `jmapgen_int` (as const reference) instead of splitting it here. There are quite a few functions that need to combine those two values (min, max) into one via `rng` later on, they could use `jmapgen_int::get()` instead.

This may also be better for the future, if (when) `jmapgen_int` gets expanded to allow other distributions.